packages/ckeditor5-dev-ci/lib/format-message.js

```diff
@@ -81,15 +81,15 @@ function getNotifierMessage( options ) {
 		return '_The author of the commit was hidden. <https://github.com/ckeditor/ckeditor5/issues/9252|Read more about why.>_';
 	}
 
+	if ( bots.includes( options.githubAccount ) ) {
+		return '_This commit is a result of merging a branch into another branch._';
+	}
+
 	// If the author of the commit could not be obtained, let's ping the entire team.
 	if ( !slackAccount ) {
 		return `@channel (${ options.commitAuthor }), could you take a look?`;
```

Comment:
```suggestion
		return `<!channel> (${ options.commitAuthor }), could you take a look?`;
```

https://api.slack.com/reference/surfaces/formatting#special-mentions

<img width="1303" alt="image" src="https://github.com/ckeditor/ckeditor5-dev/assets/2270764/25af4aab-c158-430c-87e2-8bc96cff521b">

cc: @psmyrek